openproblems/tasks/spatial_decomposition/datasets/sc_to_sp.py

```diff
@@ -0,0 +1,33 @@
+from ....data.pancreas import load_pancreas
+from ....tools.decorators import dataset
+from ._sc_to_sp_utils import generate_synthetic_dataset
+from ._sc_to_sp_utils import get_pancreas_integer
+
+# @dataset("Pancreas (alpha=1)")
+# def pancreas_alpha_1(test=False, n_obs=100):
+#     adata = load_pancreas(test=False)
+#     adata = get_pancreas_integer(adata)
+#     adata.obs["label"] = adata.obs["celltype"]
+
+#     merged_adata = generate_synthetic_dataset(adata, n_obs=n_obs, alpha=1)
+#     return merged_adata
+
+
+@dataset("Pancreas (alpha=5)")
+def pancreas_alpha_5(test=False, n_obs=100):
+    adata = load_pancreas(test=False)
+    adata = get_pancreas_integer(adata)
+    adata.obs["label"] = adata.obs["celltype"]
+
+    merged_adata = generate_synthetic_dataset(adata, n_obs=n_obs, alpha=5)
+    return merged_adata
+
+
+# @dataset("Pancreas (alpha=0.5)")
```

Comment:
Either uncomment or delete

Reply:
If they are currently not working, remove from here and open a new draft PR that will add them in and we can fix it later